workflows/testing/test_workflow/manifest.py

```diff
@@ -0,0 +1,45 @@
+import yaml
+
+class BundleManifest:
```

Comment:
We can separate this out as a separate tool to read/write manifest file

Reply:
Yup, there will be a big refactoring exercise later to unify this code with the build workflow, and pull out common components. Code re-use is not top priority right now; working prototype is the goal.